sdcclient/_monitor_v1.py

```diff
@@ -0,0 +1,288 @@
+import json
+import copy
+import requests
+import re
+
+from sdcclient._monitor import SdMonitorClient
+
+try:
+    basestring
+except NameError:
+    basestring = str
+
+
+class SdMonitorClientV1(SdMonitorClient):
```

Comment:
Do we want to maintain this code? Or we better document to use a git-tag (pointing to code before this merge) to use legacy dashboards API?

Reply:
Good question. Unfortunately I think it's hard to tell. Given that the vast majority of "monitor" client functionalities are still the same, I think that this model allows better maintainability of the overall client. I'm confident that little bug fixes may always be required, but I also see that after a "deprecation" period the older version can be easily removed (to be replaced with the next "deprecation" in line).

Does this makes sense?

Reply:
For me it makes more sense having a clean `master` branch and maintaining a `2019-04` branch. At both cases users have to deal with some complexity (knowing which git branch they have to checkout, or knowing which class to instantiate).

I see you point, we can live with that (specially if users use pypi). But I don't like having that in our beloved `master`.

Reply:
Unfortunately no option is ideal. For the moment, only a (small) portion of `SdMonitorClient` functionalities need to be split across `V1` and "latest", in addition to all the other 3 classes (`SdSecureClient`, `SdScanningClient`, `_SdcCommon`). This means that the majority of additions and changes would need to be back-ported to the separate release branch (even though some changes might not apply to "latest" version of the API). Over time, you may also end up with several release branches, potentially making the maintenance cost higher and higher.

One option that could be a tradeoff is keeping `V1` in the same branch/build for a given deprecation period, after which we can essentially abandon the older version in favor of the latest one. Especially for this first time, this would also allow us to evaluate what could be the best option moving forward (eg. if we realize that the maintenance cost is negligible). Thoughts?